server/com.eclipsesource.uml.modelserver/src/main/java/com/eclipsesource/uml/modelserver/commands/AddAssociationEdgeCommand.java

```diff
@@ -0,0 +1,58 @@
+/********************************************************************************
+ * Copyright (c) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
+ * available at https://opensource.org/licenses/MIT.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR MIT
+ ********************************************************************************/
+package com.eclipsesource.uml.modelserver.commands;
```

Comment:
We should move the Notation commands to their own Package. They will add up quickly, and it will become harder and harder to maintain proper semantic/notation separation otherwise.

Reply:
Yes I agree, I split them up into `*.commands.compound`, `*.commands.notation` and `*.commands.semantic`.